static/js/answers-experience.js

```diff
@@ -0,0 +1,51 @@
+import RuntimeConfig from './runtime-config';
+
+/**
+ * Check if Answers is initialized in window in a timed loop
+ * 
+ * @returns {Promise} When resolved, Answers exists in window
+ */
+function waitForAnswersInit() {
```

Comment:
I think we need to do more than just wait for the Answers instance to appear on the window because the setAnalyticsOptIn function depends on the `_analyticsReporterService` which isn't set until Answers initializes. One idea is that we could put a promise on the window such as `AnswersInitialized` in layouts.hbs, and in core.hbs, we can resolve that promise inside of the Answers onReady. So rather than having a `waitForAnswersInit` function, we could use `window.AnswersInitialized`

Reply:
I like this suggestion! we can even use a fancy "Answers init returns a promise" that somebody added a while back and set that as the promise's value :^)

Reply:
I didn't realize that Answers is put on window before it's initialized but yeah connor's idea sounds good! I just updated following his suggestion. I didn't see your comment beforehand oliver, what do you mean by answers init returns a promise? like in ANSWERS.init pr initAnswers()?

Reply:
Yup! ANSWERS.init() actually returns a promise. I think this was Connor's first item https://github.com/yext/answers-search-ui/pull/996